### DISCUSSION TITLE
the title displayed on the TOP screen

### DISCUSSION THREAD

**Discussion by tttll1091** [danny-avila/LibreChat#430] [Q&A]
Is there any license problem to edit and operate the title displayed on the TOP screen called ChatGPT Clone?

**Comment by danny-avila** [comment]
No license issue, and you can change it in /client/.env with VITE_APP_TITLE="your app title"

We are also officially changing the name of the project in a few days

**Comment by tttll1091** [comment]
thanks
I try
